public/docs/dart/latest/guide/component-styles.jade

```diff
@@ -1 +1,40 @@
-!= partial("../../../_includes/_ts-temp")
\ No newline at end of file
+extends ../../../ts/latest/guide/component-styles.jade
+
+block includes
+  include ../_util-fns
+
+//- TODO: consider adding material equivalent to TS Appendices 1 & 2 if relevant.
+
+block style-url-alert
+  :marked
+    The URL is ***relative to the Dart application package root*** which usually
```

Comment:
You are correct, thanks. Fixed (though there seems to be a bug related to the styles actually being picked up properly; I'll try to isolated and file an issue soon).